test/tools/isoltest.cpp

```diff
@@ -45,8 +45,8 @@ namespace fs = boost::filesystem;
 struct TestStats
 {
 	int successCount;
-	int runCount;
-	operator bool() const { return successCount == runCount; }
+	int testCount;
```

Comment:
Actually, this one should also have an initializer.

Reply:
If these are initialized, then I can't construct ``TestStats`` from an initializer list anymore without also adding a constructor to ``TestStats`` - that's why they are not initialized here...

Reply:
But I would postpone such cosmetics changes until I rework the test system in general in 0.5.1 (at least as far as my plan goes).